specification/newrelic/NewRelic.Management/examples/Monitors_GetMetricStatus_MinimumSet_Gen.json

```diff
@@ -7,12 +7,17 @@
     "resourceGroupName": "rgNewRelic",
     "monitorName": "fhcjxnxumkdlgpwanewtkdnyuz",
     "request": {
+      "azureResourceIds": [
+        "enfghpfw"
```

Comment:
>enfghpfw [](http://example.com/codeflow?start=9&length=8)

Is this suppose to be resourceId format of an ARM resource? Then could you please add example in same format? Ex: /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/........

Reply:
Please update in all the examples where ids suppose to be in arm resourceId format.

Reply:
Done. @samsaha-ms

Reply:
@vipray-ms , I see still in all arm resourceIds are arbitrary strings in example.